styx-scheduler-service/src/test/java/com/spotify/styx/StateInitializingTriggerTest.java

```diff
@@ -111,8 +104,11 @@ public void shouldInjectTriggerExecutionEventWithBackfillTrigger() throws Except
 
   @Test
   public void shouldDoNothingIfDockerInfoMissing() throws Exception {
-    Workflow workflow = Workflow.create("id", TestData.DAILY_WORKFLOW_CONFIGURATION);
-    setDockerImage(workflow.id(), workflow.configuration());
+    final WorkflowConfiguration configuration =
```

Comment:
This test was checking that nothing is done when docker args are missing, while now it checks that nothing is done when docker image is missing. Let's rename this `shouldDoNothingIfDockerImageMissing`. Maybe it's good to add a test case that tests initialization succeed even with no docker args provided.